src/decoration.cpp

```diff
@@ -149,23 +149,25 @@ void Decoration::resize_outline(Rectangle outline, const DecorationScheme& schem
 
     auto tile = inner;
     client_->applysizehints(&inner.width, &inner.height);
-    if (!false) { // formely: if (!tight_decoration)
```

Comment:
Was the if(!tight_decoration) removed on purpose?

Reply:
(For the record: long explanation in https://github.com/herbstluftwm/herbstluftwm/pull/858#issuecomment-618831793)

Reply:
Thanks!

Reply:
Thanks!